mobile/ios/ios-device.ts

```diff
@@ -54,6 +54,21 @@ export class IOSDevice implements Mobile.IiOSDevice {
 			this.deviceInfo.version = this.getValue("ProductVersion");
 			this.deviceInfo.color = this.getValue("DeviceColor");
 			this.deviceInfo.isTablet = productType && productType.toLowerCase().indexOf("ipad") !== -1;
+
+			productType = productType.toLowerCase().trim();
+			let majorVersionAsString = _.last(productType.split(",")[0]);
```

Comment:
It is better to use regex here in order to works correctly if productType is for example `iphone10,2`